go/vt/topo/zk2topo/serving_graph.go

```diff
@@ -57,9 +57,6 @@ func (zs *Server) GetSrvKeyspace(ctx context.Context, cell, keyspace string) (*t
 	if err != nil {
 		return nil, err
 	}
-	if len(data) == 0 {
```

Comment:
With proto3, an empty []byte is actually a valid encoded value. This was returning an error when it shouldn't have..